src/lib/bootloader/kexec.rb

```diff
@@ -99,8 +99,8 @@ def copy_kernel
       cmd = Yast::Builtins.sformat(
         "/bin/cp -L %1/%2 %1/%3 %4",
         Yast::Installation.destdir,
-        "vmlinuz",
-        "initrd",
+        "boot/vmlinuz",
+        "boot/initrd",
```

Comment:
Doesn't this deserve a test? As far as I see, the new test is only about the localization fix and has nothing to do with the original bugfix (these two lines).